CONTRIBUTING.md

```diff
@@ -122,7 +122,7 @@ You should also add yourself to the `CONTRIBUTORS.md` file in the root of the pr
 
 ### Merging Changes
 
-* It is considered "poor form" to merge your own request.
+* You (or another contributor) can merge your pull request after it has been approved by at least one other contributor and there are no requests for changes.
```

Comment:
I'd like to add to the end of the sentence, "and there are no unaddressed comments / unanswered questions," so that folks know it's poor form to (self-)merge when there are still comments and questions to address.